lib/v8debugapi.js

```diff
@@ -485,18 +484,20 @@ module.exports.create = function(logger_, config_, fileStats_) {
         }
       }
     }
-    var localConfig;
     if (breakpoint.action === 'LOG') {
-      localConfig = { capture: { maxFrames: 0 } };
-      _.defaultsDeep(localConfig, config);
+      var frame = execState.frame(0);
+      var evaluatedExpressions = [];
+      breakpoint.expressions.forEach(function(exp) {
```

Comment:
Could this block be replaced by `breakpoint.expressions = breakpoint.expressions.map( ... );`